src/components/AccountList.tsx

```diff
@@ -391,23 +378,23 @@ const BuildAccountTable = (
     isMobile ? (
       translateRaw(id)
     ) : (
-      <div key={id} onClick={() => updateSortingState(id)}>
-        {translateRaw(id)} <IconArrow isFlipped={getColumnSortDirection(id)} />
-      </div>
+      <Box variant="rowAlign" key={id} onClick={() => updateSortingState(id)}>
+        <Text as="span">{translateRaw(id)}</Text>
+        <Icon
+          ml="0.3ch"
+          type="sort"
+          isActive={getColumnSortDirection(id)}
+          size="1em"
+          color="linkAction"
+        />
+      </Box>
     );
 
   const columns = [
     convertColumnToClickable('ACCOUNT_LIST_LABEL'),
     convertColumnToClickable('ACCOUNT_LIST_ADDRESS'),
     convertColumnToClickable('ACCOUNT_LIST_NETWORK'),
-    <HeaderAlignment
-      key={'ACCOUNT_LIST_VALUE'}
-      align="center"
-      onClick={() => updateSortingState('ACCOUNT_LIST_VALUE')}
-    >
-      {translateRaw('ACCOUNT_LIST_VALUE')}
-      {!isMobile && <IconArrow isFlipped={getColumnSortDirection('ACCOUNT_LIST_VALUE')} />}
-    </HeaderAlignment>,
```

Comment:
Missing alignment on desktop now

Reply:
The previous sort icon alignments were all wrong. 
Currently the icon alignment is correctly placed at the center of the line-height.
<img width="1159" alt="Screenshot 2021-03-03 at 16 58 15" src="https://user-images.githubusercontent.com/2429708/109833607-b10bef80-7c41-11eb-94b2-f3782819fecd.png">

Am I missing something?

Reply:
I think the value header should be right aligned when the value itself is 😄

Reply:
Gotcha! Looking into it, it appears the `HeaderAlignments` don't actually work (for 'PRIVATE' or 'REMOVE' columns. It seems that it may require a complete refactor of AccountList for it to use the table correctly. Request to fix later.